templates/registry/markdown/event_macros.j2

```diff
@@ -9,6 +9,13 @@ The event name MUST be `{{ event.name }}`.
 {%if event.note %}
 {{ event.note | trim }}
 {% endif %}
+{%- if event.entity_associations is sequence %}
+
+The event SHOULD be associated with one of the following entities:
+{% for e in event.entity_associations %}
+- `{{ e }}`
```

Comment:
possible improvement, probably not easy to do today: would be nice to add a link to the doc where entity is defined

Reply:
absolutely agree, you saw our discussion in semconv tooling.